synadm/cli/room.py

```diff
@@ -159,3 +159,14 @@ def search_room_cmd(ctx, search_term, from_, limit):
                .format(search_term.capitalize()))
     ctx.invoke(list_room_cmd, from_=from_, limit=limit,
                name=search_term.capitalize())
+
+
+@room.command()
+@click.argument("room_id", type=str)
+@click.argument("user_id", type=str)
```

Comment:
since this is optional, make it an option. hava a look on other commands how to use @click.option decorators. best use --user-id, -u, also add help and describe the behaviour (I often copy paste and shorten stuff from the api documentation ;-)

Reply:
If you use is_flag than click will always send a boolean! So the default value will be true or false and not None! Work with that! Sorry for misleading you here!